Editor/EditorCore/EditorUtility.cs

```diff
@@ -250,11 +250,20 @@ internal static bool HasStaticFlag(this GameObject go, StaticEditorFlags flags)
         }
 
         /// <summary>
-        /// Move a GameObject to the active scene, where active scene may be a prefab stage.
+        /// Move a GameObject to the proper active root.
+        /// Checks if a default parent exists, otherwise it adds the object as a root of the active scene, which can be a prefab stage.
         /// </summary>
         /// <param name="gameObject"></param>
-        internal static void MoveToActiveScene(GameObject gameObject)
+        internal static void MoveToActiveRoot(GameObject gameObject)
```

Comment:
This method seems to be used in other classes (at least in `ShapeEditor`) but I don't see modifications in other places after this renaming.

Reply:
This could be made public as well

Reply:
@lopezt-unity good catch, was missing a file

Reply:
@karl- Might be better to make InitObject public